src/main/java/uk/gov/di/ipv/dto/CredentialIssuers.java

```diff
@@ -1,17 +1,25 @@
 package uk.gov.di.ipv.dto;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
 import java.util.Collections;
-import java.util.Objects;
 import java.util.Set;
 
 public class CredentialIssuers {
 
     private final Set<CredentialIssuerConfig> credentialIssuerConfigs;
 
+    @JsonIgnore private String source;
```

Comment:
I've just tried it and I don't think we need the `@JsonIgnore` annotation here. The object mapper will happily give us a CredentialIssuers object with this field set to null.